libcodechecker/server/client_db_access_server.py

```diff
@@ -120,10 +125,13 @@ def check_auth_in_request(self):
         return success
 
     def do_GET(self):
-        authToken = self.check_auth_in_request()
-        if authToken:
+        authSession = self.check_auth_in_request()
+        authToken = None
+        if authSession:
```

Comment:
Same comment as in `client_auth_handler`, this extra conditional should be simplified out, or better to remove the `authToken` variable and use `authSession.token` instead. A session object, if valid, always has a token, anyways.